onnxscript/tests/function_libs/torch_lib/ops_test.py

```diff
@@ -149,20 +161,24 @@ def run_test_output_match(
         op: The OpInfo instance. instantiate_device_type_tests provides this.
         function_executor: The function executor. This is a function that takes
             a function and its arguments and returns the output of the function.
+        tested_op_mapping: The mapping of op name to the tested op.
     """
     samples = op.sample_inputs(
         device,
         dtype,
         requires_grad=False,
     )
 
-    onnx_function_and_wrangler = ops_test_data.OPINFO_FUNCTION_MAPPING[op.name]
+    onnx_function_and_wrangler = tested_op_mapping[op.name]
     # Obtain the input_wrangler that manipulates the OpInfo inputs
     # to match the aten operator signature
     # An example is nn.functional.upsample_nearest2d, which has a different signature
     # than the aten operator upsample_nearest2d
     onnx_function, input_wrangler = _split_function_and_wrangler(onnx_function_and_wrangler)
-    if not ops_test_common.dtype_op_schema_compatible(dtype, onnx_function.op_schema):
+    if (
+        not ops_test_common.dtype_op_schema_compatible(dtype, onnx_function.op_schema)
+        and dtype != torch.complex64
```

Comment:
is_complex

Reply:
torch.is_complex only works on tensors. I changed this to COMPLEX_TYPES we define in the file